views/bloghome.ejs

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <% const title = ctx.prismicH.asText(bloghome.data.headline); %> 
+    <%- include('./partials/head', {title}); %>
+  </head>
+  <body>
+    <div class="home">
+      <div
+        class="blog-avatar"
+        style="background-image: url(<%-bloghome.data.image.url%>)"
+      ></div>
+      <h1 class="blog-title">
+        <%- ctx.prismicH.asText(bloghome.data.headline) %>
+      </h1>
+      <p class="blog-description">
+        <%- ctx.prismicH.asText(bloghome.data.description) %>
+      </p>
+    </div>
+    <div class="blog-main">
+      <% posts.forEach((post) => { %>
+      <div class="blog-post">
+        <a href="<%- ctx.prismicH.asLink(post) %>">
+          <h2><%- ctx.prismicH.asText(post.data.title) %></h2>
+        </a>
+        <p class="blog-post-meta">
+          <% const date = UIhelpers.getDate(post) %>
+          <span><%- date %></span>
```

Comment:
Unless you reuse `date` somewhere else in this file, it's unnecessary to save it in a const. You can just output it directly:
```
<span><%- UIhelpers.getDate(post) %></span>
```

Reply:
Okay. Made changes.